src/promptflow-devkit/promptflow/_proxy/_python_executor_proxy.py

```diff
@@ -79,8 +79,7 @@ async def exec_aggregation_async(
         aggregation_inputs: Mapping[str, Any],
         run_id: Optional[str] = None,
     ) -> AggregationResult:
-        with self._flow_executor._run_tracker.node_log_manager:
-            return self._flow_executor._exec_aggregation(batch_inputs, aggregation_inputs, run_id=run_id)
+        return self._flow_executor.exec_aggregation(batch_inputs, aggregation_inputs, run_id=run_id)
```

Comment:
The reason of removing node_log_manager is?

Reply:
Will call it inside `exec_aggregation`. Don't need to call twice.